zk_toolbox/crates/config/src/forge_interface/deploy_l2_contracts/input.rs

```diff
@@ -27,6 +30,10 @@ impl DeployL2ContractsInput {
             bridgehub: contracts.ecosystem_contracts.bridgehub_proxy_addr,
             governance: wallets.governor.address,
             erc20_bridge: contracts.bridges.erc20.l1_address,
+            consensus_registry_owner: wallets
+                .developer_multisig
```

Comment:
I'm not sure it's the correct name. 
Do we need an additional wallet, other than governor?

Reply:
I don't know, @vladbochok suggested using the "dev multisig" account/wallet. It didn't sound like any of the other wallets, so I made a guess that we need to make room for a new one to be defined, but I could be completely off the mark.

Reply:
Vlad refered to dev multisig, as a potential governor as the entity that already exists in ML structure. it doesn't mean it has to be in our tooling, because it's really specific to our self. 
I'd rather prefer to have governor as a default entity with an ability to change the governor to whatever we prefer later

Reply:
Okay great, removed.